File: CSharp/Q3. Max Consecutive Ones.cs
public class Solution {
    public int FindMaxConsecutiveOnes(int[] nums) {
         int max = 0, current = 0;

         for(int i =0;i<nums.Length; i++)
        {
            if(nums[i] == 1)
            {
                current++;

                if(current > max)
                max=current;
            }
            else
            {
                current = 0;
            }
        }

        return max;
    }
}

File: TypeScript/Q3. Max Consecutive Ones.ts
function FindMaxConsecutiveOnes(nums: number[]): number {
    let max: number = 0, current: number = 0;

    for (let i = 0; i < nums.length; i++) {
        if (nums[i] === 1) {
            current++;
            if (current > max) max = current;
        } else {
            current = 0;
        }
    }

    return max;
}
